changelog.json

```
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "3.1.2-beta1": [
+      "[Added] You can preview the changes a pull request from your current branch would make - #11517",
+      "[Fixed] App correctly remembers undo commit prompt setting - #15408",
+      "[Improved] Added support for zooming out at the 67%, 75%, 80% and 90% zoom levels - #15401. Thanks @sathvikrijo!",
```

[Per our docs](https://github.com/desktop/desktop/blob/development/docs/process/writing-release-notes.md#present-tense) this should use present tense:
```suggestion
      "[Improved] Add support for zooming out at the 67%, 75%, 80% and 90% zoom levels - #15401. Thanks @sathvikrijo!",
```